src/client/gui/argparser.h

```diff
@@ -13,22 +13,24 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
- * Authored by: Alberto Aguirre <alberto.aguirre@canonical.com>
- *
  */
 
-#ifndef MULTIPASS_CLI_H
-#define MULTIPASS_CLI_H
+#ifndef ARGPARSER_H
+#define ARGPARSER_H
```

Comment:
Don't we always have "MULTIPASS_" at the start of our #ifdefs?

Reply:
Right, I was just blindly following the cli's `ArgParser` header which does this:)  But yeah, will fix it to follow the convention.

Reply:
Fixed